### PR TITLE
Added IPv6 tests for tacacs_server and tacacs_server_group

### PR DESCRIPTION
**Pull request by jonnytpuppet**

### CODE REVIEW

**Comment by jonnytpuppet** [comment]
tacacs_server:
  TestCase :: tacacs_server Resource :: All Attributes Defaults :: PASS
  TestCase :: tacacs_server Resource :: All Attributes Defaults :: PASS
TestCase :: tacacs_server Resource :: All Attributes Defaults :: End
tests/beaker_tests/tacacs_server/tacacs_server_provider_defaults.rb passed in 53.22 seconds
      Test Suite: tests @ 2016-02-01 15:32:46 +0000




tacacs_server_group:
  TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: PASS
  TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: PASS
TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: End
tests/beaker_tests/tacacs_server_group/tacacs_server_group_provider_defaults.rb passed in 52.41 seconds
      Test Suite: tests @ 2016-02-01 15:34:14 +0000

**Comment by glennmatthews** [comment]
:+1:

File: tests/beaker_tests/tacacs_server_group/tacacs_server_group_provider_defaults.rb [hunk 1]
```diff
@@ -62,18 +62,14 @@
...
-    on(agent, cmd_str, acceptable_exit_codes: [0, 2])
+    resource_absent_cleanup(agent, 'tacacs_server')
+    resource_absent_cleanup(agent, 'tacacs_server_group')
 
     cmd_str = get_vshell_cmd('conf t ; tacacs-server host 2.2.2.2')
     on(agent, cmd_str)
     cmd_str = get_vshell_cmd('conf t ; tacacs-server host 3.3.3.3')
     on(agent, cmd_str)
-    cmd_str = get_vshell_cmd('conf t ; tacacs-server host 4.4.4.4')
+    cmd_str = get_vshell_cmd('conf t ; tacacs-server host 2004::44')
```
Comment (mikewiebe) [inline tests/beaker_tests/tacacs_server_group/tacacs_server_group_provider_defaults.rb]: We are trying to move away from using vsh commands to configure the device.  This does not work well across our various Cisco platforms.  Can we rework this setup section to use `tacacs_server` or `cisco_tacacs_server_host` instead?

**Comment by jonnytpuppet** [comment]
@mikewiebe code updated

**Comment by jonnytpuppet** [comment]
TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: PASS
  TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: PASS
TestCase :: tacacs_server_group Resource :: All Attributes Defaults :: End
tests/beaker_tests/tacacs_server_group/tacacs_server_group_provider_defaults.rb passed in 53.03 seconds
      Test Suite: tests @ 2016-02-02 14:15:05 +0000

**Comment by mikewiebe** [comment]
:+1:

**Pull request merged**